haystack/components/rankers/diversity.py

```diff
@@ -0,0 +1,238 @@
+import logging
```

Comment:
The failing tests are due to how we've changed how we do logging in Haystack. The fix is easy, just remove the above line so remove `import logging` and then add `from haystack import logging` which you can append to the same import list on line 4.

Reply:
Thanks! I will update the imports to fix this.